INSTALL.md

```diff
@@ -393,12 +393,12 @@ To run just the markdown benchmarks:
 [NixOS]: https://nixos.org/nixos/packages.html
 [Slackware]: https://www.slackbuilds.org/result/?search=pandoc&sv=
 [Ubuntu]: https://packages.ubuntu.com/pandoc
-[download button]: https://github.com/jgm/pandoc/releases/latest {.btn .btn-block .btn-primary}
```

Comment:
This is valid pandoc markdown, and it's needed to make the download link on <http://pandoc.org/installing.html> a big, visible button.  GitHub markdown doesn't understand this, though, so it won't render right on GitHub.  If we have to choose, I'd say it's more important for it to render correctly on pandoc.org.

We could replace the markdown link with some raw HTML, though I don't know if this would survive GitHub's rendering chain either...

Reply:
Another option would be to change this to INSTALL.txt, so GitHub won't attempt to render it as Markdown.

Reply:
hey there! thanks for the feedback. makes sense: i found the document via github and obviously thought the sole source of truth was living there, but had no idea it was rendered elsewhere.

happy to consider whichever other option, just thought it was a quick win (but obviously not so simple!)